dataactbroker/handlers/fileHandler.py

```diff
@@ -1793,20 +1797,25 @@ def serialize_submission(submission):
     files = get_submission_files(jobs)
     status = get_submission_status(submission, jobs)
     certified_on = get_lastest_certified_date(submission)
+    window_end = get_window_end(submission)
+    time_period = get_time_period(submission)
     agency_name = submission.cgac_agency_name if submission.cgac_agency_name else submission.frec_agency_name
     return {
-        "submission_id": submission.submission_id,
-        "last_modified": str(submission.updated_at),
-        "status": status,
-        "agency": agency_name if agency_name else 'N/A',
-        "files": files,
+        'submission_id': submission.submission_id,
+        'last_modified': str(submission.updated_at),
+        'status': status,
+        'agency': agency_name if agency_name else 'N/A',
+        'files': files,
         # @todo why are these a different format?
-        "reporting_start_date": str(submission.reporting_start_date) if submission.reporting_start_date else None,
-        "reporting_end_date": str(submission.reporting_end_date) if submission.reporting_end_date else None,
-        "user": {"user_id": submission.user_id, "name": submission.name if submission.name else "No User"},
-        "certifying_user": submission.certifying_user_name if submission.certifying_user_name else "",
+        'reporting_start_date': str(submission.reporting_start_date) if submission.reporting_start_date else None,
+        'reporting_end_date': str(submission.reporting_end_date) if submission.reporting_end_date else None,
+        'user': {'user_id': submission.user_id, 'name': submission.name if submission.name else 'No User'},
+        'certifying_user': submission.certifying_user_name if submission.certifying_user_name else '',
         'publish_status': PUBLISH_STATUS_DICT_ID[submission.publish_status_id],
-        "certified_on": str(certified_on) if certified_on else ""
+        'certified_on': str(certified_on) if certified_on else '',
+        'quarterly_submission': submission.is_quarter_format,
+        'window_end': str(window_end) if window_end else '',
```

Comment:
Is there a reason we're passing the actual end of the window instead of just "Passed" or "Upcoming"?

Reply:
My logic was that the frontend could use `today()` and if they wanted to, they also have access to the end date if they wanted to list that on the frontend too. I'm fine with either.